views/endorsement-backers.js

```diff
@@ -0,0 +1,103 @@
+const { html } = require('../helpers')
+const stateNames = require('datasets-us-states-names-abbr')
+
+const legislatorTitles = {
```

Comment:
There are more legislators than just California and Dane County... We should use the `name` or `short_name` fields in the database so the naming is consistent and works for every legislator.

Reply:
Agreed. 
This was just a temp fix to match existing Proof docs.
There is a placeholder (generic `Legislator`) in there for places without enumerated titles, but agree would be better to be grabbing this field from the API, ideally in the `offices[]` object.